projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/components/before-you-start/before-you-start.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { Router } from '@angular/router';
+import { ErrorMessage } from '../../../../../domain';
+import { LinkedCasesState } from '../../domain';
+import { LinkedCasesPages } from '../../enums';
+import { LinkedCasesService } from '../../services/linked-cases.service';
+
+@Component({
+  selector: 'ccd-linked-cases-before-you-start',
+  templateUrl: './before-you-start.component.html'
+})
+export class BeforeYouStartComponent {
+
+  @Output()
+  public linkedCasesStateEmitter: EventEmitter<LinkedCasesState> = new EventEmitter<LinkedCasesState>();
+
+  public isLinkCasesJourney = false;
+
+  public errorMessages: ErrorMessage[];
+
+  constructor(private readonly router: Router, public readonly linkedCasesService: LinkedCasesService) {
```

Comment:
@johnbenjamin-hmcts - the `Router` injection is marked as `private` but the `LinkedCasesService` injection is `public`... any reason for the difference?

Reply:
@Daniel-Lam - It was odd to keep it public to access in the HMTL. I have done the code changes accordingly.